src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -92,8 +116,13 @@ def handle_result_for_project(
                     extra={"project_id": project_subscription.project_id, **result},
                 )
 
-            if result["status"] == CHECKSTATUS_FAILURE:
-                create_issue_platform_occurrence(result, project_subscription)
+            if project_subscription.mode == ProjectUptimeSubscriptionMode.AUTO_DETECTED_ONBOARDING:
+                self.handle_result_for_project_auto_onboarding_mode(project_subscription, result)
+            elif project_subscription.mode in (
+                ProjectUptimeSubscriptionMode.AUTO_DETECTED_ACTIVE,
+                ProjectUptimeSubscriptionMode.MANUAL,
```

Comment:
Does this work or do you need to do `.value`?

Reply:
Yeah with `IntEnum` we don't need to use `.value`

Reply:
python is the worst 😂